pinephone/vibrator/include/vibrator-impl/FFDevice.h

```diff
@@ -0,0 +1,41 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (C) 2019 The Android Open-Source Project
+ * Copyright (C) 2021 GloDroid project
+ */
+
+#pragma once
+
+#include <dirent.h>
```

Comment:
Please remove unused includes, and move includes that is required by implementation to .cpp file